CommonApi/src/main/java/mezz/jei/api/registration/JeiRegistrationStep.java

```diff
@@ -0,0 +1,33 @@
+package mezz.jei.api.registration;
+
+/**
+ * The order of values in this enum is not guaranteed. Use {@code ordinal()} with caution, if ever.
+ * @see JeiRegistrationDelegate
+ */
```

Comment:
all new stuff should have `@since #.#.#` in the javadoc. you can bump the version number in the gradle properties